funnel/views/commentvote.py

```diff
@@ -99,42 +100,41 @@ def new_comment(self):
                     parent = Comment.query.filter_by(
```

Comment:
This one has been tripping me every time I look at it. I keep thinking the "parent" is the proposal, but it's the parent comment. We've made this mistake in the model itself where the column is called "parent_id". We should:

1. Rename the "parent" variable here to "in_reply_to_comment"
2. In a separate PR, rename the column to "in_reply_to_id" and replace all references to "parent" with "in_reply_to" (the term "In-Reply-To" is an email header to indicate a reply, with the reference message id.)

Reply:
Please file a ticket for the rename.